src/locales/en/ui.yml

```diff
@@ -50,6 +53,10 @@ component:
   watch:
     streamingFor: Streaming for {0}
     nextUpInMugen: Next Up
+    Comments:
```

Comment:
Maybe keep the format of just `comment.all: all`, not sure if it's needed to be a yaml list